src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -171,7 +171,9 @@ export class PiaIntakeService {
     }
     if (query.filterByMinistry) {
       whereClause.forEach((clause) => {
-        clause.ministry = query.filterByMinistry;
+        clause.ministry = mpoMinistries.includes(query.filterByMinistry)
```

Comment:
This won't work for non-MPO drafters. 
And, It won't work for MPOs who drafted a PIA for a ministry they are not MPO of.

Reply:
the drafter can not see filter by ministry dropdown list

Reply:
Agree. That's today. Please drop a comment stating this assumption for non-MPO users.

However, what about MPOs who draft a PIA for a ministry they are not MPO of scenario?

Reply:
verified. It works for both the scenarios.